internal/routing/acquire.go

```diff
@@ -48,21 +47,27 @@ func checkAcquired(w http.ResponseWriter, r *http.Request, ls *lockservice.Simpl
 		return
 	}
 
-	var req lockservice.LockRequest
+	var req lockservice.LockCheckRequest
 	err = json.Unmarshal(body, &req)
 
 	if err != nil {
 		http.Error(w, err.Error(), http.StatusBadRequest)
 		return
 	}
 
-	desc := &lockservice.SimpleDescriptor{
+	desc := &lockservice.LockDescriptor{
 		FileID: req.FileID,
-		UserID: req.UserID,
 	}
-	if ls.CheckAcquired(desc) {
-		w.Write([]byte("checkAcquire success"))
+
+	owner, ok := ls.CheckAcquired(desc)
+	if ok {
+		byteData, err := json.Marshal(lockservice.CheckacquireRes{Owner: owner})
```

Comment:
seriously. here lsCheck**A**cquired is called but the object is called Check**a**cquireRes

why??

Reply:
Oops check the final comment xD

Reply:
@SUMUKHA-PK  does this need to be rectified?

Reply:
Done